Code/Editor/GameEngine.cpp

```diff
@@ -565,6 +566,11 @@ void CGameEngine::SwitchToInGame()
         streamer->QueueRequest(flush);
         wait.acquire();
     }
+
+    if (gSettings.clearConsoleOnGameModeStart)
+    {
+        CConsoleSCB::GetCreatedInstance()->ClearText();
```

Comment:
I think this will crash if the user had closed the Console. It might be better to change it so the Console listens for the eNotify_OnBeginGameMode event and then clears itself then, so you don't have to add that new ClearText API as well.

Reply:
That's nicer, changed.